apps/codeforafrica/src/components/FooterNavList/index.js

```diff
@@ -0,0 +1,65 @@
+import { Link } from "@commons-ui/next";
+import { Grid } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function FooterNavList({ menu }) {
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <Grid
+      component="nav"
+      item
+      xs={12}
+      md={3}
+      sx={{
+        order: { xs: 1, md: 2 },
+        display: "flex",
+        justifyContent: "center",
+      }}
+    >
```

Comment:
You shouldn't start a component with `<Grid item`. You're basically "leaking" the parent structure to the child component. Let parent handle its own ### <Grid container/`<Grid item` pairs. If you need a `Grid` in a child component, fine, start with `<Grid container`.

Basically `Grid` must be self contained in each component.